<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">


	<head>
		<title>Marta Sestelo</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="stylesheet" href="academicons/academicons.css"/>
		<link rel="stylesheet" href="font-awesome/css/font-awesome.css"/>  
		<meta name="description" content="Marta Sestelo's personal page.">
		<meta name="keywords" content="data science, statistics, uvigo, uminho, University of Vigo, University of Minho, regression, nonparamteric, R, Fortran">
		<meta name="author" content="Marta Sestelo"> 
		<!-- <meta name="MobileOptimized" content="320">   -->
		<meta charset="utf-8">
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta http-equiv="Content-Style-Type" content="text/css" href="style.css" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
	</head>


	<body>
		<h3><a href="http://sestelo.github.io">sestelo.github.io</a></h3>

		<div class="group-icons">
			<a href="https://github.com/sestelo/sestelo.github.io/blob/master/images/sestelo_photo.jpg"><img id="foto" src="https://github.com/sestelo/sestelo.github.io/blob/master/images/sestelo_photo2.png?raw=true" alt="foto"></a>
			<a href="https://github.com/sestelo?tab=repositories"><i class="fa fa-github fa-2x"></i></a>
			<a href="http://orcid.org/0000-0003-4284-6509"><i class="ai ai-orcid fa-2x"></i></a> 
			<a href="https://www.researchgate.net/profile/Marta_Sestelo"><i class="ai ai-researchgate-square fa-2x"></i></a>
			<a href="https://www.linkedin.com/pub/marta-sestelo/5a/289/12"><i class="fa fa-linkedin-square fa-2x"></i></a>
			<a href="https://twitter.com/MartaSestelo"><i class="fa fa-twitter-square fa-2x"></i></a>	
		</div>




		<div id="navbar">
			<a href="01_aboutme.html"><strong>About me</strong></a> &nbsp; &nbsp; &nbsp; 
			<a href="02_research.html"><strong>Research</strong></a> &nbsp; &nbsp; &nbsp; 
			<a href="03_teaching.html"><strong>Teaching</strong></a> &nbsp; &nbsp; &nbsp; 
			<a href="04_software.html"><strong>Software</strong></a> &nbsp; &nbsp; &nbsp; 
			<a href="05_news.html"><strong>News</strong></a>
		</div>


		<p><br> <br> <br> <br> <br> <br></p>


		<div id="contact2">
			<a href="mailto:sestelo@uvigo.es">sestelo@uvigo.es</a><br /><a href="http://sestelo.github.io">sestelo.github.io</a> <br /> <a href="http://sidor.uvigo.es">SiDOR group, Uvigo</a><br />
		</div>

		<div id="contact1">
			Marta Sestelo<br/><a href="http://www.cmat.uminho.pt/Default.aspx?lang=en-US">Centre of Mathematics</a><br />University of Minho<br />
		</div>




		<div id="texto">


			<h2><a name="software"></a>Software</h2>
			<ul>
				<li><strong>clustcurv</strong> package. Author of this R package (<a href="https://github.com/noramvillanueva/clustcurv/">https://github.com/noramvillanueva/clustcurv/</a>) for clustering curves with an automatic selection of their number. It is focused on both regression and survival curves, using the k-means or k-medians.</li>

				<p><br></p>
				
				<li><strong>survidm</strong> package. Author and maintainer of this R package (<a href="http://cran.r-project.org/web/packages/survidm/">http://cran.r-project.org/web/packages/survidm/</a>) which implement some newly developed methods for the estimation of several probabilities in an illness-death model. The package can be used to obtain nonparametric and semiparametric estimates for: transition probabilities, occupation probabilities, cumulative incidence function and the sojourn time distributions. Several auxiliary functions are also provided which can be used for marginal estimation of the survival functions.</li>

				<p><br></p>
				
				<li><strong>condSURV</strong> package. Author and maintainer of this R package (<a href="http://cran.r-project.org/web/packages/condSURV/">http://cran.r-project.org/web/packages/condSURV/</a>) to implement some newly developed methods for the estimation of the conditional survival function. The package implements three nonparametric and semiparametric estimators for these quantities. It also implements feasible estimation methods for these quantities conditionally on current or past covariate measures. Other related estimators are also implemented in the package. One of these estimators is the Kaplan-Meier estimator typically assumed to estimate the survival function. A modification of the Kaplan-Meier estimator, based on a preliminary estimation (presmoothing) of the censoring probability for the survival time, given the available information is also implemented.</li>

				<p><br></p>

				<li><strong>npregfast</strong> package. Author and maintainer of this R package (<a href="http://cran.r-project.org/web/packages/npregfast/">http://cran.r-project.org/web/packages/npregfast/</a>) to perform nonparametric estimation for analyzing interactions factor-by-curve. npregfast allows the user to obtain nonparametric estimates using local linear kernel smoothers and compare them between factor’s levels. Also a feature of the package is its ability to draw inference about critical points, such as maxima or change points linked to the derivative curves. The inference (confidence intervals and tests) is based on bootstrap. This package allows not only to obtain smooth estimates also based on classical parametric models, as allometric model, one of the most used models in biology frameworks usually used to study the relationship between two biometrical variables. Additionally, we have implemented binning type acceleration techniques.</li>

				<p><br></p>

				<li><strong>FWDselect</strong> package. Author and mantainer of this package (<a href="http://cran.r-project.org/web/packages/FWDselect/">http://cran.r-project.org/web/packages/FWDselect/</a>), an R package that introduces a simple method to select the best model or best subset of variables using different types of responses (gaussian, binary or poisson) and applying it in different contexts (parametric or nonparametric).</li>

				<p><br></p>
				
				<li><strong>seq2R</strong> package. Author of this R package (<a href="http://cran.r-project.org/web/packages/seq2R">http://cran.r-project.org/web/packages/seq2R</a>) to detect compositional changes in genomic sequences. This software is useful for loading .fasta or .gbk files, and for retrieving sequences from GenBank dataset. The package allows to detect differences or asymmetries based on nucleotide composition by using local linear kernel smoothers. Also, it is possible to draw inference about critical points (i. e. maximum or minimum points) related with the derivative curves. Additionally, bootstrap methods have been used for estimating confidence intervals and speed computational techniques (binning techniques) have been implemented in <tt>seq2R<tt>.</li>
			</ul>
			<p><br></p>
			<hr />

		</div>


		<div id="copyright">
		© <a href="http://sestelo.github.io">Marta Sestelo</a>, 2017
		</div>


		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-41678546-2', 'auto');
		  ga('send', 'pageview');

		</script>



	</body>


</html>








<!--


</body>
-->




